docs/security/trust/trust_sandbox.md

```diff
@@ -61,146 +63,114 @@ work exclusively with the Docker Hub, you would not need with these components.
 They are built into the Docker Hub for you. For the sandbox, however, you must
 build your own entire, mock production environment and registry.
 
-### Configure /etc/hosts
-
-The sandbox' `notaryserver` and `sandboxregistry` run on your local server. The
-client inside the `notarysandbox` container connects to them over your network.
-So, you'll need an entry for both the servers in your local `/etc/hosts` file.
-
-1. Add an entry for the `notaryserver` to `/etc/hosts`.
-
-        $ sudo sh -c 'echo "127.0.0.1 notaryserver" >> /etc/hosts'
-
-2. Add an entry for the `sandboxregistry` to `/etc/hosts`.
-
-        $ sudo sh -c 'echo "127.0.0.1 sandboxregistry" >> /etc/hosts'
-
-
 ### Build the notarytest image
 
-1. Create a `notarytest` directory on your system.
+1. Create a `trustsandbox` directory on your system and change into it.
 
-        $ mkdir notarysandbox
+        $ mkdir trustsandbox
+        $ cd trustsandbox
 
-2. Change into your `notarysandbox` directory.
+2. Create a filed called `Dockerfile` with your favorite editor.  For example, using vim:
 
-        $ cd notarysandbox
+        $ touch Dockerfile
+        $ vim Dockerfile
 
-3. Create a `notarytest` directory then change into that.
+3. Add the following to the new file.
 
-        $ mkdir notarytest
-        $ cd notarytest
+        FROM docker:dind
 
-4. Create a filed called `Dockerfile` with your favorite editor.
-
-5. Add the following to the new file.
-
-        FROM debian:jessie
-
-        ADD https://master.dockerproject.org/linux/amd64/docker /usr/bin/docker
-        RUN chmod +x /usr/bin/docker \
-          && apt-get update \
-          && apt-get install -y \
+        RUN apk add --no-cache \
           tree \
           vim \
           git \
-          ca-certificates \
-          --no-install-recommends
+          ca-certificates
 
-        WORKDIR /root
-        RUN git clone -b trust-sandbox https://github.com/docker/notary.git
-        RUN cp /root/notary/fixtures/root-ca.crt /usr/local/share/ca-certificates/root-ca.crt
+        ADD https://raw.githubusercontent.com/docker/notary/master/fixtures/root-ca.crt /usr/local/share/ca-certificates/root-ca.crt
         RUN update-ca-certificates
 
-        ENTRYPOINT ["bash"]
+        ENTRYPOINT ["dockerd-entrypoint.sh"]
+        CMD []
```

Comment:
This `ENRYPOINT` and `CMD` is already defined in the `docker:dind` image, so are not needed here (see https://github.com/docker-library/docker/blob/866c3fbd87e8eeed524fdf19ba2d63288ad49cd2/1.11/dind/Dockerfile)